src/client/styles/style.css

```diff
@@ -6,28 +6,34 @@ a {
   text-decoration: none;
 }
 
-a#basket {
+a#cart {
   padding: 0 0 0 18px;
   position: relative;
   top: 7px;
 }
 
-a#basket::after {
-  background-color: red;
-  border-radius: 50%;
+a#cart::after {
+  /* background-color: red; */
```

Comment:
There are quite a few commented lines. Do they need to stay or can they just be deleted?

Reply:
Good point — got rid of it.

(It was stuff that we might want back: sticky header, circular red badge, etc.)